provided-template.html

```diff
@@ -0,0 +1,453 @@
+    <!-- Styles used through the page-->
+    <style>
+
+    @import url('https://fonts.googleapis.com/css?family=Fira+Sans');
+
+      h2 {
+        color: #387C2C;
+        font-weight:500;
+        font-size: 27px;
+        font-family: 'Fira Sans', sans-serif;
+        margin:0px;
+        line-height: 32px;
+      }
+      h3 {
+        color:#444444;
+        text-transform: uppercase;
+        font-weight: 600;
+        font-size: 18px;
+        font-family: 'Fira Sans', sans-serif;
+      }
+      h4 {
+        font-size: 15px;
+        font-family: 'Fira Sans', sans-serif;
+        color: white;
+        font-weight: 500;
+        line-height: 10px;
+      }
+      .align-left {
+        text-align:left;
+      }
+      p {
+        color: #606160;
+        font-weight: 300;
+        font-family: 'Fira Sans', sans-serif;
+        font-size:14px;
+        line-height: 26px;
+        margin:0px;
+      }
+      .read-more a {
+        text-decoration: none;
+        font-size:13px;
+        font-weight: 600;
+        color: #347928;
+        font-family: 'Fira Sans', sans-serif;
+      }
+      .border-top {
+        border-top: 1px solid #CBCCCB;
+      }
+      .tags {
+        font-style: italic;
+        font-family: 'Fira Sans', sans-serif;
+      }
+      .position {
+        color: white;
+        font-size: 14px;
+        font-weight: 300;
+        font-style: normal;
+      }
+      .name {
+        margin: 12px 0px 0px 0px;
+        font-weight: 400;
+        font-style: normal;
+      }
+      .events p {
+        line-height: 20px;
+      }
+      .fa {
+        font-size: 30px;
+        text-align: right;
+        padding-top:20px;
+      }
+      .social-icons {
+        color: #444444;
+        height: 36px;
+        width: 36px;
+        margin: 0em;
+        padding: 0em;
+        display: inline-block;
+        vertical-align: text-top;
+        border: 1px solid #444444;
+        text-align: center;
+        border-radius: 50%;
+        line-height: 36px;
+        font-size: 21px;
+        margin-right:25px;
+        font-weight: 600;
+        padding-top:0px;
+      }
+      .social-icons:hover {
+        color: #387C2C;
+      }
+      .social-bar {
+        text-align: center;
+      }
+      .social-bar-links {
+        text-decoration: none;
+        color: #858585;
+        margin-top:20px;
+      }
+      .social-bar-links:hover {
+        color: #387C2C;
+      }
+
+    </style>
+
+    <table align="center" border="0" cellpadding="0" cellspacing="0" width="960" bgcolor="white">
+<!-- 1th Row Header-->
+      <tr>
+        <td align="center" bgcolor="#3b7b30">
+          <a href="https://www.pausd.org/"><img src="images/pausd.svg" alt="PAUSD LOGO" width="300" height="180" style="display: block;"></a>
+          <h4 style="text-align:left;margin:-35px 0px 0px 40px;position:absolute;font-weight: 300;">NOVEMBER 2016</h4>
```

Comment:
Don't use `<heading>` tags for formatting, they should describe the data, in this case the DOM, and should come in order, and should typically come in order. For example, `<h2>` tags should only come after an `<h1>` tag.

Heading tags are used to describe subject headings and subheadings. Using the heading tags like this is helpful for SEO, and accessibility (things like screen readers).

I feel like this `<h4>` is only here to formatting (larger, heavier font with padding). Don't do this.